pyfr/backends/openmp/kernels/par-gimmik.mako

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can we unify with par-xsmm?

Reply:
I prefer keeping these in separate files.

Reply:
Having XSMM and GiMMiK functions both have the same signatures would reduce code duplication.  It would also be nice for pairing as all matrix multiplications would take a uniform code path/argument set.